src/elm/TreeUtils.elm

```diff
@@ -88,6 +88,14 @@ getColumn n tree =
   ListExtra.getAt n cols
 
 
+getColumnById : String -> Tree -> Maybe Column
+getColumnById id tree =
+  let
+    n = getDepth 0 tree id
+  in
+  getColumn n tree
```

Comment:
I added this function for you to make it easier to get the current column, given a card's id.

Reply:
This will be very helpful. I spent some time trying to come up with a good way of getting this, but I think I was conflating groups/columns along the way, so the results I was seeing were, unsurprisingly, unpredictable.